docs/source/index.rst

```diff
@@ -20,15 +20,25 @@ on data-structures in the NVMe specification documents.
 To read more about what **Spex** does, see :ref:`sec-what-is-spex`.
 For help on setting up **Spex** on your system, see :ref:`sec-setup`.
 
-For direct usage of nvme-spex as a linting tool it is possible to run it from
-docker. For setup of docker on windows we refer to guides from `Docker Desktop
+
+For direct usage of nvme-spex it is possible to run it from docker. For setup of
+docker on windows we refer to guides from `Docker Desktop
 <https://docs.docker.com/desktop/install/windows-install/>`_ or `Podman
```

Comment:
I think the reference to **Podman** should be replace with a reference to **Podman Desktop**: https://podman-desktop.io/downloads On this topic, why propose **Docker Desktop**? I don't see any value of **Docker Desktop** over **Podman Desktop**, more choice usually turns into confusion.